scripts/core/editor3/reducers/suggestions.jsx

```diff
@@ -670,13 +746,21 @@ const setMergeParagraphSuggestion = (editorState, data) => {
  */
 const setDeleteSuggestionForCharacter = (editorState, data) => {
     let selection = editorState.getSelection();
-    const currentStyle = Highlights.getHighlightStyleAtOffset(editorState, changeSuggestionsTypes, selection, -1);
-    const currentData = currentStyle != null ? Highlights.getHighlightData(editorState, currentStyle) : null;
 
     if (selection.getStartOffset() === 0) {
         return setMergeParagraphSuggestion(editorState, data);
     }
 
+    const paragraphStyle = Highlights.getHighlightStyleAtOffset(editorState, paragraphSuggestionTypes, selection, -1);
+
+    if (paragraphStyle != null) {
+        // if current character is marked as paragraph suggestion, skip
```

Comment:
It's a bit late to comment on this, but it seems to me that it's a bad idea to implement `split-paragraph-suggestion` by inserting a character into the text itself. Inline styles can be put on this character, you can comment and add annotations on it. I feel it would be a constant edge-case in all work related to editor3

Reply:
This is the agreed solution with POs. Let's see how it works and we will see if we will change or not.
Right now the only problem I see is when add a comment or an annotation on this single character. The comment/annotation will be deleted on paragraph suggestion accept/reject. I will fix it if it will be needed.